compass/landice/tests/ismip6_run_ais/ismip6_run_ais.cfg

```diff
@@ -0,0 +1,44 @@
+# config options for ismip6 antarctic ice sheet data set
+[ismip6_run_ais]
+
+# Resolution that will be set up.  Should be one of 08, 04 with leading 0.  The paths below must manually be set to be consistent
+mesh_res = 08
+
+# COMPASS load script path
+# This is user-specific.
+compass_load_path = /global/cscratch1/sd/hoffman2/AIS8km/compass/load_dev_compass_1.2.0-alpha.1_cori-haswell_gnu_mpt_albany.sh
+
+# Base path to the processed input ismip6 ocean and smb forcing files. User has to supply.
+forcing_basepath = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/forcing/ais_mesh_8to30km_res
+
+# Path to the initial condition file. User has to supply.
+init_cond_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027/relaxation_0TGmelt_10yr_muCap/AIS_8to30km_r01_20220906.smooth3.basinsFineTuned_carvedRonne_CIRWIP_relaxation_0TGmelt_10yr_muCap.nc
+
+# Path to the file for the basal melt parametrization coefficients.
+melt_params_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027/basin_and_coeff_gamma0_DeltaT_quadratic_non_local.nc
```

Comment:
Shouldn't this be in the `forcing_basepath` and a user should only need to supply a string like the following?
```
melt_params = quadratic_non_local
```

Reply:
In our tuning, we ended up recalibrating deltaT manually and not using the default files created by the forcing.  We want the flexibility to do that, though in some cases one might choose to use the default ocean params file.